source/configure/plugins-configuration-settings.rst

```diff
@@ -980,6 +980,26 @@ Enable AV1 (Experimental)
 | :ref:`simulcast is enabled for screen sharing <configure/plugins-configuration-settings:enable simulcast for screen sharing (experimental)>`.                          |
 +--------------------------------------------------------------------------+---------------------------------------------------------------------------------------------+
 
+Enable DC signaling (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. config:setting:: plugins-enabledcsignaling
```

Comment:
At some point on this page, search meta data started being positioned below the section heading, rather than directly above the section header. It's my understanding that search meta data needs to be directly above the section heading to return results correctly.

Reply:
Thanks, not sure either, but it should look better now.